app/modules/adengine-assets.js

```diff
@@ -0,0 +1,61 @@
+import {Promise} from 'rsvp';
+
+export const assetUrls = {
+	adEngineScript: '/mobile-wiki/assets/adengine/ad-engine.global.js',
+	adProductsScript: '/mobile-wiki/assets/adengine/ad-products.global.js',
+	geoScript: '/mobile-wiki/assets/adengine/geo.global.js',
+	adProductsStyles: '/mobile-wiki/assets/adengine/ad-products.css'
```

Comment:
We already have styles added:
https://github.com/Wikia/mobile-wiki/blob/d22c6623abd692fe45453721c4730d4d6a653f72/app/styles/module/_ads.scss#L4

Reply:
huh, ofc :)